opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -178,60 +160,87 @@ def get_handle(self, label_values: List[str]) -> "object":
         a value was not provided are permitted.
 
         Args:
-            label_values: A list of label values that will be associated
-                with the return handle.
+            label_values: Values to associate with the returned handle.
         """
 
-    def remove_handle(self, label_values: List[str]) -> None:
+    def remove_handle(self, label_values: Tuple[str, ...]) -> None:
```

Comment:
Do we need such a method? What are the use cases?

Reply:
This was left from the old specs that originated from Java. In the new specs proposal, there is no mention of this so I'll remove it.